src/CalculatorUnitTests/DateCalculatorUnitTests.cpp

```diff
@@ -657,7 +659,7 @@ TEST_METHOD(DateCalcViewModelInitializationTest)
 
 TEST_METHOD(DateCalcViewModelAddSubtractInitTest)
 {
-    auto viewModel = ref new DateCalculatorViewModel();
+    DateCalculatorViewModel ^ viewModel = ref new DateCalculatorViewModel();
```

Comment:
nit: prefer using "auto" like it was before. Same feedback for DateCalcViewModelDateDiffDaylightSavingTimeTest